lib/std/os/linux/io_uring.zig

```diff
@@ -636,6 +636,51 @@ pub const IO_Uring = struct {
             @ptrCast(*const c_void, fds.ptr),
             @intCast(u32, fds.len),
         );
+        try convert_registration_error(res);
+    }
+
+    /// Registers the file descriptor for an eventfd that will be notified of completion events on
+    //  an io_uring instance.
+    pub fn register_eventfd(self: *IO_Uring, fd: os.fd_t) !void {
+        assert(self.fd >= 0);
+        comptime assert(@sizeOf(os.fd_t) == @sizeOf(c_int));
```

Comment:
This check looks like it can be removed given `fd_t` is [defined as `i32`](https://github.com/ziglang/zig/blob/master/lib/std/os/bits/linux.zig#L41) which should also be the [same as `c_int`](https://www.ibm.com/docs/en/ibm-mq/9.2?topic=platforms-standard-data-types-aix-linux-windows) on linux platforms.

Reply:
I agree, but it is already there on `reigster_files`, should I remove it on both?

Reply:
Sure.